packages/core/src/components/steps-modal/steps-modal.tsx

```diff
@@ -23,6 +23,8 @@ export class AtomStepsModal {
   @Prop() primaryText?: string
   @Prop() secondaryText?: string
   @Prop() closeOnFinish?: boolean
+  @Prop() disablePrimary?: boolean
```

Comment:
it is missing the storybook's stories of those guys, right?

Reply:
done at [508ed2d](https://github.com/juntossomosmais/atomium/pull/609/commits/508ed2dd19c46b1a866caf7985d684b3db0b2205)